ckanext/datagovuk/tests/test_paths.py

```diff
@@ -19,8 +20,10 @@ def test_harvest_path(self):
         self.assertEqual(path, '/harvest')
 
     def test_healthcheck_path(self):
-        path = toolkit.url_for('healthcheck')
-        self.assertEqual(path, '/healthcheck')
+        app = helpers._get_test_app()
+        with app.flask_app.test_request_context():
```

Comment:
:thinking:

ah yes, this is because these tests don't actually test any view/request in particular, they merely want to unit-test some internal machinery (in this case `url_for`) which needs to be inside a request context to work properly.

Reply:
We could still use `app.get` for consistency?

Reply:
What url would we be `.get`ting in this case then?

Reply:
ok...

The test is still to specifically test that the `url_for` call works from within a flask request context. So I'm using the context manager to ensure it's called from within a flask request context.

Reply:
ok, lets not block this PR just on this issue, we can move to the more consistent way when we have done the migrated over to flask fully.